packages/galpy/meta.yaml

```diff
@@ -0,0 +1,36 @@
+package:
+  name: galpy
+  version: 1.7.2
+source:
+  url: https://files.pythonhosted.org/packages/67/7f/ae16e887e78d42c8db0406ca7eec651502ebf3e1c9782a00f59e506a8db3/galpy-1.7.2.tar.gz
+  sha256: 4011b57b427d791addd8201754989a8f08d46925ee81cd571473d414ee38c04a
+  patches:
+    - patches/0001-Force-GSL-version-to-2.7-for-pyodide-build.patch
+    - patches/0002-Don-t-use-OpenMP-for-pyodide-build.patch
+build:
+  cflags: |
+    -I$(PYODIDE_ROOT)/packages/libgsl/build/libgsl-2.7
+  ldflags: |
+    -L$(PYODIDE_ROOT)/packages/libgsl/build/libgsl-2.7/.libs
+    -L$(PYODIDE_ROOT)/packages/libgsl/build/libgsl-2.7/cblas/.libs
```

Comment:
Please use `pkg-config` for this. (See packages/msprime/meta.yaml.)

Reply:
Ah, will do! I was looking at the `msprime` PR, didn't notice that there had been updates...